TraineeTasks/StringExtension.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace TraineeTasks
+{
+    public static class StringExtension
+    {
+        public static int GetNumberOccurrences(this string str, string substr)
+        {
+            if (String.IsNullOrEmpty(substr))
+            {
+                throw new ArgumentException(String.Format("{0} - cannot be null or emty", nameof(substr)));
+            }
+
+            return Regex.Matches(str, substr).Count;
```

Comment:
For this task Regex is not required.
Regex are heavy for the CPU, use it only if really needed. Here you can you Split method to solve task

Reply:
Done